app/src/main/java/ai/doc/netrunner/activities/ModelManagerActivity.kt

```diff
@@ -23,14 +33,18 @@ class ModelManagerActivity : AppCompatActivity(), ModelBundleListFragment.Callba
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_model_manager)
 
-        modelBundlesViewModel.setBundleManagers(TIOModelBundleManager(applicationContext, ""), TIOModelBundleManager(applicationContext, ""))
-
-        val toolbar = findViewById<Toolbar>(R.id.toolbar)
-        setSupportActionBar(toolbar)
+        modelBundlesViewModel.setBundleManagers(
+                TIOModelBundleManager(applicationContext, ""),
+                TIOModelBundleManager(ModelManagerUtilities.getModelFilesDir(this)))
 
+        setSupportActionBar(findViewById<Toolbar>(R.id.toolbar))
```

Comment:
Looks good. Just a suggestion, we can try view-binding too.

https://developer.android.com/topic/libraries/view-binding

Reply:
https://github.com/doc-ai/net-runner-android/issues/70